include/ck_tile/ops/fmha/kernel/fmha_bwd_tile_partitioner.hpp

```diff
@@ -7,38 +7,34 @@
 
 namespace ck_tile {
 
-template <typename BlockFmhaShape_>
-struct FmhaBwdTilePartitioner
+template <ck_tile::index_t kN0>
+struct FmhaBwdKTilePartitioner
 {
-    using BlockFmhaShape = ck_tile::remove_cvref_t<BlockFmhaShape_>;
-
-    static constexpr ck_tile::index_t kN0 = BlockFmhaShape::kN0;
-
     CK_TILE_HOST static constexpr auto
     GridSize(ck_tile::index_t batch_size_, ck_tile::index_t nhead_, ck_tile::index_t seqlen_k_)
     {
         // TODO: this may need tuning
-        return dim3(ck_tile::integer_divide_ceil(seqlen_k_, kN0), nhead_, batch_size_);
+        return dim3(batch_size_, nhead_, ck_tile::integer_divide_ceil(seqlen_k_, kN0));
     }
 
     CK_TILE_DEVICE auto operator()(ck_tile::index_t /*seqlen_k*/)
     {
```

Comment:
Suggest to remove the using of `FmhaBwdTilePartttioner`: 

1. Using Partitioner class and use it as kernel template will probably lead to more kernel instances and compiling-time
2. It does not make sense to let the host codes to implicitly set the `kN0` template for defining the Partitioner instance 
3. For `GridSize()` and `Operator()`,   using specific arguments is not better than using `Const Kargs *pKarg`

So remove `Partitioner` and define `GridSize(const Kargs *pArgs)` and  `GetTileIndex(const Kargs *pArgs)`directly in the kernel

Reply:
OK.